databallpy/load_data/metadata.py

```diff
@@ -24,3 +24,25 @@ class Metadata:
     away_players: pd.DataFrame
     away_score: int
     away_formation: str
+
+    def __eq__(self, other):
+        result = (
+            self.match_id == other.match_id
+            and self.pitch_dimensions == other.pitch_dimensions
+            and self.match_start_datetime == other.match_start_datetime
+            and self.periods_frames.equals(other.periods_frames)
+            and self.frame_rate == other.frame_rate
+            if not pd.isnull(self.frame_rate)
+            else pd.isnull(other.frame_rate)
```

Comment:
Dit geeft True als : 
-frame_rates hetzelfde zijn
-beide NaN zijn
Begrijp ik het dan goed?

Reply:
En deze functie had je nodig om voor het testen?

Reply:
In python comparisons with np.nan are always false: `np.nan == np.nan` results in `False`. The extra check is needed to compare if both values are np.nan.

Reply:
Yes, by default if you would test `Metadata == Metadata` it would check every attribute with the `==` comparison. However, the example you pointed out and when checking if pd dataframes are alike it doesn't work out. So I had to overwrite the method.